### PR TITLE
[v6w8Sc5Z] Bump neo4j and apoc versions

### PR DESCRIPTION
**Pull request by nadja-muller**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -72,8 +72,8 @@ subprojects {
         // neo4jDockerImage system property is used in TestContainerUtil
         systemProperties 'user.language' : 'en' ,
                 'user.country ' : 'US',
-                'neo4jDockerImage' : project.hasProperty("neo4jDockerVersionOverride") ? 'neo4j:' + project.getProperty("neo4jDockerVersionOverride") + '-enterprise-debian' : 'neo4j:4.4.24-enterprise-debian',
-                'neo4jCommunityDockerImage': project.hasProperty("neo4jDockerVersionOverride") ? 'neo4j:' + project.getProperty("neo4jDockerVersionOverride") + '-debian': 'neo4j:4.4.24-debian'
+                'neo4jDockerImage' : project.hasProperty("neo4jDockerVersionOverride") ? 'neo4j:' + project.getProperty("neo4jDockerVersionOverride") + '-enterprise-debian' : 'neo4j:4.4.25-enterprise-debian',
+                'neo4jCommunityDockerImage': project.hasProperty("neo4jDockerVersionOverride") ? 'neo4j:' + project.getProperty("neo4jDockerVersionOverride") + '-debian': 'neo4j:4.4.25-debian'
```
Comment (ncordon) [inline build.gradle:76]: This should still be 4.4.24 I think

Comment (nadja-muller) [reply build.gradle:76]: Because the docker release is lagging behind?

Comment (ncordon) [reply build.gradle:76]: Ah no, that's not the problem, have you updated the secrets?

**Review by ncordon — APPROVED** [review]

File: NOTICE.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Neo4j
-Copyright © 2002-2023 Neo4j Sweden AB (referred to in this notice as "Neo4j")
+Copyright � 2002-2023 Neo4j Sweden AB (referred to in this notice as "Neo4j")
```
Comment (ncordon) [inline NOTICE.txt:2]: This change is weird?

**Review by ncordon — CHANGES_REQUESTED** [review]
It looks good except for the NOTICE file header

**Review by ncordon — APPROVED** [review]

**Pull request merged**